syne_tune/optimizer/schedulers/synchronous/hyperband.py

```diff
@@ -290,6 +290,9 @@ def _suggest(self, trial_id: int) -> Optional[TrialSuggestion]:
             config = self.searcher.get_config(trial_id=str(trial_id))
             if config is not None:
                 config = cast_config_values(config, self.config_space)
+                self.searcher.register_pending(
```

Comment:
are these changes related to the feature?

Reply:
Yes, this is something that was missing and came to light because of the new test